WebRoot/js/zimbraAdmin/servers/controller/ZaServerController.js

```diff
@@ -239,8 +243,9 @@ function (params) {
 						excludeMasks.push(cidrData);
 						
 						for(var j=(numIFs-1);j>=0;j--) {
-							if(ZaIPUtil.isInSubNet(cidrData, locals2[j].ipData)) {
-								throw new AjxException(AjxMessageFormat.format(ZaMsg.ERROR_LOCAL_ADDR_EXCLUDED,[locals2[j].ipData.src, chunks[i]]),AjxException.INVALID_PARAM,"ZaServerController.prototype.validateMyNetworks");
+							//if(ZaIPUtil.isInSubNet(cidrData, locals2[j].ipData)) {
```

Comment:
better to remove code if not in use

Reply:
Hi @grishick , @silentsakky. I updated the code. Please review and tell me if I need to update anything else